manim/mobject/types/vectorized_mobject.py

```diff
@@ -49,6 +49,8 @@
 
 
 class VMobject(Mobject):
+    """A vectorized mobject."""
```

Comment:
Can you do this in a new PR?

Reply:
I'm sorry, actually it is in another PR but I made a mistake and added that code here too. Here's the PR with these changes #1749
I just deleted these changes in this PR